packages/cms-cli/commands/create.js

```diff
@@ -125,6 +140,10 @@ function configureCreateCommand(program) {
         case TYPES['website-theme']:
           createTheme(dest, type, program);
```

Comment:
TODO: Refactor `website-theme` to use `createProject` which needs a special case to only copy `/src`